docs/source/events/examples/payment-status-change.yaml

```diff
@@ -17,5 +17,17 @@ currency_code: EUR
 # Payment method specific data (stripe intent ID in this case)
 payment_data:
   transaction_id: pi_3NOKIODvbS6ezf4w21zIR6Ft
+# Line items:
+line_items:
+- name: one-off
+  amount: 10
```

Comment:
This is an integer not a float? If I'm right that we don't support donation of £2.50, then we could say that so developers know they can treat this as an integer

Reply:
At the moment our forms don’t allow non-integer values. The value here is a “float encoded as string“ (see `index.rst`).

If we specify here that this can only be integers then introducing floats in IST 2.0 forms would be a breaking change.